composer.json

```diff
@@ -1,51 +1,48 @@
 {
     "name": "silverstripe/demo.silverstripe.org",
     "description": "Demo SilverStripe Project",
+    "license": "BSD-3-Clause",
     "require": {
-        "php": ">=8.1.0",
-        "silverstripe/installer": "^5",
-        "silverstripe/dynamodb": "^5",
-        "silverstripe/crontask": "^3"
+        "silverstripe/installer": "^6",
+        "silverstripe/dynamodb": "^6",
+        "silverstripe/crontask": "^4",
+        "symfony/process": "^7.3"
     },
     "config": {
         "process-timeout": 600,
         "allow-plugins": {
-            "composer/installers": true,
             "silverstripe/recipe-plugin": true,
-            "silverstripe/vendor-plugin": true
+            "silverstripe/vendor-plugin": true,
+            "composer/installers": true,
+            "phpstan/extension-installer": true,
+            "dealerdirect/phpcodesniffer-composer-installer": true
+        },
+        "platform": {
+            "php": "8.3"
         }
```

Comment:
```suggestion
```
Having this means we have to update the PHP version in two places instead of just one

Reply:
Is the other place you're referring to the the PHP 8.3 in platform.yml? 

This is in composer.json to ensure that when composer.lock is generated on a developer machine it installs PHP 8.3 compatible deps so that it works on the server it's being deployed to i.e. doesn't install PHP 8.4 / 8.5 deps that work on the developer machine because that's what they're running on their local